helm.cr

```
@@ -142,6 +142,28 @@ module Helm
     [resp[:status].success?, output_file]
   end
 
+  def self.generate_manifest(release_name : String, namespace : String)
+    Log.info { "Generating manifest from installed CNF: #{release_name}" }
+
+    helm = BinarySingleton.helm
+    cmd = "#{helm} get manifest #{release_name} --namespace #{namespace}"
+    Log.info { "helm command: #{cmd}" }
+
+    status = Process.run(cmd, shell: true, output: output = IO::Memory.new, error: stderr = IO::Memory.new)
+    
+    if status.success?
+      Log.debug { "Helm.manifest output:\n #{output.to_s}" }
+      Log.info { "Manifest was generated successfully" }
+      if !stderr.empty?
+        Log.info { "Helm.manifest stderr: #{stderr.to_s}" }
+      end
+    else
+      Log.error { "There is some problem with generating the manifest: #{stderr.to_s}" }
+      raise "Helm.manifest stderr: #{stderr.to_s}"
+    end
```

It would likely be better to raise a specific exception, you can create a custom exception class that inherits the Exception class, this would make catching the exception in the main testsuite code more readable.